### PR TITLE
Updates for the release

### PR DESCRIPTION
**Pull request by rwcarlsen**
This should be merged as part of the final 1.4 release process.  Note that the release notes are not entirely complete yet - the cycamore and cymetric sections still ne a bit of work.

### CODE REVIEW

**Comment by gonuke** [comment]
Needs addition for `tolerance_*` variables

**Comment by rwcarlsen** [comment]
I'd be happy to take PR's into my branch here to keep this up to date, or someone can borrow my commits and build as needed on top in their own new PR - whichever is preferred on the UW side.

File: source/previous/v1.4.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+--------------------------------------------------
+
+* Added Storage archetype.  TODO details
+ 
+* Added Mixer archetype.  TODO details
+
+`Cymetric <https://github.com/cyclus/cymetric>`_
+--------------------------------------------------
+
+TODO
```
Comment (Baaaaam) [inline source/previous/v1.4.rst]: @opotowsky could you fill the the cymetric part ?

File: source/arche/testing.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 
 .. _testing:
 
-Testing Agents and Modules
-==========================
+Testing and Debuging Archetypes
```
Comment (opotowsky) [inline source/arche/testing.rst]: -> debugging

File: source/previous/v1.4.rst (hunk 1 above)
Comment (opotowsky) [inline source/previous/v1.4.rst]: Of course!

**Comment by Baaaaam** [comment]
This has been included in the #206 PR.
Thx @rwcarlsen. closing

**Pull request closed without merging**